cli/hzn.go

```diff
@@ -90,6 +90,12 @@ Environment Variables:
 	cliutils.Opts.Verbose = app.Flag("verbose", msgPrinter.Sprintf("Verbose output.")).Short('v').Bool()
 	cliutils.Opts.IsDryRun = app.Flag("dry-run", msgPrinter.Sprintf("When calling the Horizon or Exchange API, do GETs, but don't do PUTs, POSTs, or DELETEs.")).Bool()
 
+	envCmd := app.Command("env", msgPrinter.Sprintf("Show the Horizon Environment Variables."))
```

Comment:
I thought what we were doing with this fix was to display what the env vars resolved to on every CLI command? Am I forgetting what we discussed?

Reply:
Ivan reminds me this is what we discussed.